pyproject.toml

```diff
@@ -13,7 +13,7 @@ package-mode = false
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4.0"
-Django = "5.0.3"
+Django = "5.0.4"
```

Comment:
After updating Django to 5.0.4, ensure to test all Django-dependent features for compatibility, especially with major version updates. [important]